h2/test_h2_perf.py

```diff
@@ -0,0 +1,98 @@
+"""
+TLS perf tests - load Tempesta FW with multiple TLS handshakes.
+"""
+
+from framework import tester
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2020 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+NGINX_CONFIG = """
+pid ${pid};
+worker_processes  auto;
+
+events {
+    worker_connections   1024;
+    use epoll;
+}
+
+http {
+    keepalive_timeout ${server_keepalive_timeout};
+    keepalive_requests ${server_keepalive_requests};
+    sendfile         on;
+    tcp_nopush       on;
+    tcp_nodelay      on;
+
+    open_file_cache max=1000;
+    open_file_cache_valid 30s;
+    open_file_cache_min_uses 2;
+    open_file_cache_errors off;
+
+    # [ debug | info | notice | warn | error | crit | alert | emerg ]
+    # Fully disable log errors.
+    error_log /dev/null emerg;
+
+    # Disable access log altogether.
+    access_log off;
+
+    server {
+        listen        ${server_ip}:8000;
+
+        location / {
+            add_header Connection keep-alive;
```

Comment:
The header is not required at all. More over it's gives a hint about connection lifetime and response can lie about it if a request asks about connection close. Doesn't really affect anything because `tls-perf` doesn't send requests.